### ISSUE TITLE
Graphana ALert Status Change

### ISSUE THREAD

**Issue by rasShemi** [grafana/grafana#16462]
Based on for duration specified in condition query, alert status changes from pending state to alerting state. Does the alert state changes back to Ok status if the data values is below the threshold value specified?

**Comment by daniellee** [comment]
Yes - you can read about the `For` concept here: http://docs.grafana.org/alerting/rules/#for

> If an alert rule has a configured For and the query violates the configured threshold it will first go from OK to Pending. Going from OK to Pending Grafana will not send any notifications. Once the alert rule has been firing for more than For duration, it will change to Alerting and send alert notifications.
